### PR TITLE
Update MRTK and MROpenXR packages

### PR DESCRIPTION
**Pull request by keveleigh**
Updates to MRTK 2.7.0-preview.5 and MROpenXR 0.9.5.
Also updates to Unity's 1.2.0 OpenXR package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,57 +15,64 @@ products:
...
-such as Unity packages and images. Please install the latest git-lfs before cloning this repo.
+> such as Unity packages and images. Please install the latest git-lfs before cloning this repo.
 
 > NOTE: Features using AR Raycasts and AR Planes require the latest preview OpenXR runtime,
-which can be enabled [using the OpenXR Developer Tools for Windows Mixed Reality](https://docs.microsoft.com/en-us/windows/mixed-reality/develop/native/openxr-getting-started#using-preview-extensions).
-To install this app, search for "OpenXR" in the Microsoft Store app on HoloLens 2.
+> which can be enabled [using the OpenXR Developer Tools for Windows Mixed Reality](https://docs.microsoft.com/windows/mixed-reality/develop/native/openxr-getting-started#using-preview-extensions).
+> To install this app, search for "OpenXR" in the Microsoft Store app on HoloLens 2.
 
 Recommend to run these samples on HoloLens 2 using following version of tools:
```
Comment (yl-msft) [inline README.md:30]: Can you add a ## to make this a title that can be linked?  suggest:

** Recommend tools versions

**Review by yl-msft — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (keveleigh) [reply README.md]: Added!

File: README.md [hunk 2]
```diff
@@ -15,57 +15,66 @@ products:
...
+It's recommended to run these samples on HoloLens 2 using the following versions:
 
 - Latest Visual Studio 2019
-- Unity 2020.3.6f1 LTS, or later
-- Unity OpenXR plugin 1.1.1, or later
-- Mixed Reality OpenXR plugin 0.9.3, or later
-- MRTK Unity 2.7.0-preview.4, or later
+- Unity 2020.3.8f1 LTS, or later
+- Unity OpenXR plugin 1.2.0, or later
+- Mixed Reality OpenXR plugin 0.9.4, or later
```
Comment (KBonds) [inline README.md:37]: Should this be 0.9.5?

Comment (yl-msft) [reply README.md:37]: How about this?
- Latest Unity 2020.3 LTS, recommend 2020.3.6f1 or above.
- Latest Unity OpenXR plugin, recommend 1.2 or later
- Latest Mixed Reality OpenXR Plugin, recommend version 0.9.5 or later
- Latest MRTK, recommend version 2.7-preview.5 or later

Comment (keveleigh) [reply README.md:37]: Good catch! Updated

**Pull request merged**